sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py

```diff
@@ -850,7 +865,7 @@ def begin_analyze_actions(  # type: ignore
             supported languages in Text Analytics API.
         :keyword bool show_stats: If set to true, response will contain document level statistics.
         :keyword int polling_interval: Waiting time between two polls for LRO operations
-            if no Retry-After header is present. Defaults to 30 seconds.
```

Comment:
Sorry!! again I can't comment on it, but for the `param:actions` description... is it ok to say  `inputted` and `outputted` instead of `input documents` and `output documents`? idk for me it sounds weird but it could be me and English :P

Reply:
I think both are correct, but I can drop the past tense

Reply:
English :(